src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -0,0 +1,74 @@
+﻿@ECHO OFF
+SETLOCAL
+
+SET PACKAGE_DIR=%1
+SET EXECUTION_DIR=%2
+
+IF DEFINED PACKAGE_DIR ( echo Using %PACKAGE_DIR% as folder for resolving package dependencies.) ELSE (
+echo Please specify an package source directory using PACKAGEROOT parameter
+goto ShowUsage
+)
+
+if "%EXECUTION_DIR%" == "" (set EXECUTION_DIR=%~dp0)
+echo Executing in %EXECUTION_DIR% 
+
+:: ========================= BEGIN Test Execution ============================= 
+:RunTests
+
+:: Copy over dependencies... 
+GOTO CopyFiles
+:FileCopySucceeded
+
+set CORE_ROOT=core_root
+
+echo Running tests... Start time: %TIME%
+echo Command: [[TestRunCommand]]
+pushd %EXECUTION_DIR%
+call [[TestRunCommand]]
+popd
+echo Finished running tests.  End time=%TIME%, Exit code = %ERRORLEVEL%
+EXIT /B %ERRORLEVEL%
+:: ========================= END Test Execution =================================
+
+:: ========================= BEGIN Copying files  =============================== 
+:CopyFiles
+
+echo Hard linking dependent files... 
+:: Format here is: call :copyandcheck Path1 Path2 || GOTO EOF
+[[CopyFilesCommands]]
+
+IF /I "%CD%" NEQ "%EXECUTION_DIR%" (
```

Comment:
What cases do you expect the current directory to not be execution dir? I'm not sure what we are catching here.